fem/gridfunc.cpp

```diff
@@ -702,13 +1033,13 @@ int GridFunction::GetFaceVectorValues(
    {
       Transf = fes->GetMesh()->GetFaceElementTransformations(i, 4);
       Transf->Loc1.Transform(ir, eir);
-      GetVectorValues(Transf->Elem1No, eir, vals, tr);
+      GetVectorValues(*Transf->Elem1, eir, vals, &tr);
```

Comment:
This change introduces a bug: the mask `4` should become `1+4`. I found this because it is causing GLVis to crash -- `Transf->Elem1` is `NULL`.

There is a similar issue in the `else` block below.

I'll add a fix in #1529.